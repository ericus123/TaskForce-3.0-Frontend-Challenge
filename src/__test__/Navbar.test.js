/* eslint-disable no-undef */
import React from "react";
import { shallow,configure} from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import { Navbar } from "react-bootstrap";
import toJson from "enzyme-to-json";


configure({adapter: new Adapter()});
it("renders navbar without crashing", () => {

    let wrapper = shallow(<Navbar />);
    expect(toJson(wrapper)).toMatchSnapshot();
});

