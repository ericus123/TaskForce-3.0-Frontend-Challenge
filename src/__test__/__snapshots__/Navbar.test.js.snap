// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders navbar without crashing 1`] = `
<ContextProvider
  value={
    Object {
      "bsPrefix": "navbar",
      "expanded": false,
      "onToggle": [Function],
    }
  }
>
  <ContextProvider
    value={[Function]}
  >
    <nav
      className="navbar navbar-expand navbar-light"
    />
  </ContextProvider>
</ContextProvider>
`;
